doc/build.rst

```diff
@@ -161,6 +161,24 @@ To build with MinGW, type:
 
   cp make/mingw64.mk config.mk; make -j4
 
+Opening a can of worms - building XGBoost library for python for Windows with MinGW-w64
+---------------------------------------------------------------------------------------
+
+Windows versions of python are built with Microsoft Visual Studio. This has multiple issues:
```

Comment:
"Windows versions of python are built with Microsoft Visual Studio, raising several potential concerns."